packages/@lwc/integration-karma/test/events/focus-event-related-target/index.spec.js

```diff
@@ -6,6 +6,7 @@ import Container from 'x/container';
 // So this test may "flap" during local development if your browser window isn't focused (e.g. you're focused on
 // the DevTools instead).
 it('should retarget relatedTarget', function () {
+    expect(document.hasFocus(), 'Browser window must be focused for test to work').toBe(true);
```

Comment:
Web Test Runner runs tests in multiple tabs in a shared browser context. This makes focus flaky (may work in isolation, but not as part of a group) and causes tests to time out. This is not immediately obvious from the test failures, so this new assertion tries to make it more clear.